cardinal/system/system_inventory_system.go

```diff
@@ -118,3 +118,67 @@ func Take(tokens []string, playerID uint32, roomID uint32, ts *TokeniserSystem,
 
 	return description, tok_err
 }
+
+func Drop(tokens []string, playerID uint32, roomID uint32, ts *TokeniserSystem, world cardinal.WorldContext) (string, uint8) {
+	var tok_err uint8
+	var hasBeenDrop bool
+	var tok string
+	var description string
+	tok = tokens[1]
+
+	// Get the room
+	room, err := GetRoom(types.EntityID(roomID), world)
+	if err != nil {
+		world.Logger().Error().Msgf("Error getting the room with ID: %v, for the Inventory System %v", roomID, err)
+	}
+
+	// Get the player
+	player, err := GetPlayer(types.EntityID(playerID), world)
+	if err != nil {
+		world.Logger().Error().Msgf("Error getting the player wiht ID: %v, for the Inventory System: %v", playerID, err)
+	}
+
+	// Get the Object Type
+	ObjectType := ts.GetObjectType(tok)
+
+	if ObjectType != enums.ObjectTypeNone {
+		for _, lookingObject := range player.Inventory {
```

Comment:
I am being picky here but lookingObject could be changed to inventoryItem

Reply:
done